libs/@local/graph/migrations/graph-migrations/v009__entities/up.sql

```diff
@@ -77,7 +77,7 @@ CREATE TABLE entity_has_left_entity (
     FOREIGN KEY (left_web_id, left_entity_uuid) REFERENCES entity_ids
 );
 CREATE INDEX entity_has_left_entity_source_idx
-    ON entity_has_left_entity(web_id, entity_uuid);
```

Comment:
I am not sure if I am a huge fan of the removal of indentation here (or citus one), but no strong opinions really.

Reply:
Probably this is configurable. I'll take a Quick Look

Reply:
Found for `JOIN` but not for creating indices/extensions. Also not a strong opinion, so I'll just merge it 🙂